.yamato/yamato-ci.yml

```diff
@@ -0,0 +1,14 @@
+publish:
+  name: 'Build and publish Windows 7za Stevedore artifacts, including source (GPL)'
+  agent:
+    type: Unity::VM
+    image: cds-ops/win10-katana-agent
```

Comment:
Why use a Katana agent image for a Yamato job?

Reply:
I'm open to suggestions, if you know of another image with VS2017 C++ installed.

Reply:
Tried `cds-ops/win10-vs2017` ?

Reply:
Right. That one has VS2017, but doesn't have Python for the script that creates the artifact files.

Reply:
I'm not too concerned about long-term support for the Bokken image. Given the interval between us building 7za so far, any image named `*-vs2017` will almost certainly be out of date by then, anyway.

So I'm leaning towards just pushing this, and we can look at available images the next time we need to rebuild 7za, if `cds-ops/win10-katana-agent` is obsolete then.

Reply:
Alrighty.